### PR TITLE
ENCD-4278 Submitter-saved carts beta

### PR DESCRIPTION
**Pull request by keenangraham**
Permission modification to make cart object editable by owner but viewable by everyone.

### CODE REVIEW

File: src/encoded/static/components/cart/clear.js [hunk 1]
```diff
@@ -92,16 +92,17 @@ const mapStateToProps = (state, ownProps) => ({
     user: ownProps.sessionProperties,
 });
 const mapDispatchToProps = (dispatch, ownProps) => ({
-    onClearCartClick: elementAtIds => dispatch(removeMultipleFromCartAndSave(elementAtIds, ownProps.sessionProperties.user, ownProps.sessionProperties.admin, ownProps.fetch)),
+    onClearCartClick: elementAtIds => dispatch(removeMultipleFromCartAndSave(elementAtIds, ownProps.sessionProperties.user, !!(ownProps.session && ownProps.session['auth.userid']), ownProps.fetch)),
```
Comment (hitz) [inline src/encoded/static/components/cart/clear.js]: I don't understand why this particular replacement of admin->logged in is negated.

Comment (hitz) [reply src/encoded/static/components/cart/clear.js]: oh only admins are allowed to clear>?

**Comment by forresttanaka** [comment]
The admin flag is in session_properties, but checking for logged-in state for anyone comes from session. Since none of this relies on being an admin anymore, I changed the object I used to decide whether people can do cart stuff or not.

File: src/encoded/static/components/cart/clear.js (hunk 1 above)
Comment (forresttanaka) [reply src/encoded/static/components/cart/clear.js]: It’s negated negated, so positive. It’s just a trick to convert something that’s truthy to a real boolean. In this case it probably isn’t required, but I tend to like to have real booleans where booleans are expected — a style thing. I never require this of others, but tend to do that myself.

**Comment by keenangraham** [comment]
Closing in favor of https://github.com/ENCODE-DCC/encoded/pull/2525

**Pull request closed without merging**